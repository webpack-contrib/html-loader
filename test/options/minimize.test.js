/* eslint-disable */
import webpack from '../helpers/compiler';

describe('Options', () => {
  describe('minimize', () => {
    test('{Boolean}', async () => {
      const config = {
        loader: {
          test: /\.html$/,
          options: {
            minimize: true,
          },
        },
      };

      const stats = await webpack('options/minimize/fixture.js', config);
      const { source } = stats.toJson().modules[1];

      expect(source).toMatchSnapshot();
    });
  });
});
