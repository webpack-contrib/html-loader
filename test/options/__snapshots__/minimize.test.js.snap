// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Options minimize {Boolean} 1`] = `
"// HTML URLs
import HTML__URL__0 from './file.png';

// HTML
export default \`<!DOCTYPE html><html lang=\\"en\\"><head><meta charset=\\"utf-8\\"><title>HTML Loader</title></head><body><div id=\\"app\\"></div><img src=\\"\${HTML__URL__0}\\"></body></html>\`"
`;
